backend/src/backend/app.py

```diff
@@ -115,6 +131,49 @@ async def get_tasks(user_id: Annotated[str, Depends(get_user_id)]):
     return [{"id": 1, "title": "Buy Milk"}, {"id": 2, "title": "Buy Bread"}]
 
 
+# Open AI chat
+openai.api_key = OPENAI_API_KEY 
+
+
+# Pydantic models for request and response
+class ChatRequest(BaseModel):
+    message: str = Field(..., min_length=1)
+
+
+class ChatResponse(BaseModel):
+    response: str
+
+
+@app.post("/api/chat", response_model=ChatResponse, status_code=status.HTTP_200_OK)
+async def chat(chat_request: ChatRequest):
+    user_message = chat_request.message
+
+    # Prepend instruction to prompt for a brief response
+    prompt = f"Please respond in a brief phrase: {user_message}"
+
+    response = openai.chat.completions.create(
+        model="gpt-3.5-turbo",
+        messages=[
+                {"role": "user", "content": prompt}
+            ],
+        max_tokens=25,  # Limit the number of tokens in the response
+        stop=["\n"],     # Optionally, specify a stop sequence
+    )
+
+    chat_message = response.choices[0].message.content
+    print(chat_message)
+    return ChatResponse(response=chat_message)
+
+
+# override the default status for malformed request payload 422 -> 400
+@app.exception_handler(RequestValidationError)
```

Comment:
can we move this and cors middleware settings right after `app.add_middleware`? relevant code (FastAPI configuration this case `app`) should be gathered to one place.